.github/ISSUE_TEMPLATE/feature_proposal.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature proposal
+about: Propose a great new feature for Mitiq
+title: ''
+labels: 'enhancement'
+assignees: ''
+
+---
+## Pre-Request Checklist
+
+- [ ] I checked to make sure that this feature has not already been requested
+- [ ] I or my organization would be willing to help implement and/or test this.
```

Comment:
Should this be required? It seems reasonable for someone to request a feature but they can't work on it for whatever reason.

Reply:
I was just trying to emulate the format of other issue templates we have, see : https://github.com/unitaryfund/mitiq/blob/master/.github/ISSUE_TEMPLATE/bug_report.md 

It need not be required for us, I assume the volume of proposals we will get will not likely run into duplicates :)